keps/sig-network/563-dual-stack/README.md

```diff
@@ -1392,60 +1433,57 @@ This capability will move to stable when the following criteria have been met.
+  We did manual testing of a cluster turning it off and on to explore
+  disabled-with-data behavior. Testing details can be seen in [Dual-stack
+  testing](https://github.com/kubernetes/kubernetes/blob/master/test/integration/dualstack/dualstack_test.go).
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  Enabling this without configuring the CLI options will not change any default
+  behavior.
 
 ### Monitoring Requirements
 
 * **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
 
   Operators can determine if the feature is in use by listing services that 
-  employ dual-stack. This can be done via 
+  employ dual-stack. This can be done via:
 
   ```
-  kubectl get services --all-namespaces spec.ipFamilyPolicy!=SingleStack
+  kubectl get services --all-namespaces -ogo-template='{{range .items}}{{.spec.ipFamilyPolicy}}{{"\n"}}{{end}}' | grep -v SingleStack
   ```
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  Dual-stack networking is a functional addition, not a service with SLIs.
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  At a high level, this usually will be in the form of "high percentile of SLI
-  per day <= X". It's impossible to provide comprehensive guidance, but at the very
-  high level (needs more precise definitions) those may be things like:
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99,9% of /health requests per day finish with 200 code
+  N/A
 
 * **Are there any missing metrics that would be useful to have to improve observability 
 of this feature?**
-  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-  implementation difficulties, etc.).
+
+  1. For services:
+
+       Services are not in the path of pod creation. Thus, any malfunction or
+       bugs in services will not affect pods.
+
+       Services/Endpoint selection is not in path of pod creation. It runs in
+       kube-controller-manager, thus this is N/A.
+
+  2. For pods:
+
+       Dual-stack components are not in path of pod creation. It is in the path
+       of reporting pod ips. So pod creation will not be affected; if it is
+       affected, then it is a CNI issue.
```

Comment:
So, does CNI have any input from the kubelet as to whether Pods should be single or dual stack? Or is that completely out-of-band configuration that doesn't come from this feature at all? How does CNI know today which family of address to allocate?